### ISSUE TITLE
Not allowed to load local resource: blob://null/

### ISSUE THREAD

**Issue by jean-francois-labbe** [rails/importmap-rails#79]
I'm not sure if it is the right place to report this issue.

I'm seeing those errors under **safari** : 
```
[Error] Not allowed to load local resource: blob://null/assets/turbo.min.js-12ab1481b54e9f84d4561ac0e6e5328811dc5b233b7cba05444fcdee85912e46.map
[Error] Not allowed to request resource
[Error] Cannot load blob://null/assets/turbo.min.js-12ab1481b54e9f84d4561ac0e6e5328811dc5b233b7cba05444fcdee85912e46.map due to access control checks.
[Error] Not allowed to load local resource: blob://null/assets/stimulus.min.js-5cdf38f474c7d64a568a43e5de78b4313515aa0e4bd3d13fac297fffeba809f0.map
[Error] Not allowed to request resource
[Error] Cannot load blob://null/assets/stimulus.min.js-5cdf38f474c7d64a568a43e5de78b4313515aa0e4bd3d13fac297fffeba809f0.map due to access control checks.
```

where does that null is coming from ?

**Comment by dhh** [comment]
It's an issue with ES Module Shims. Tracking it here: https://github.com/guybedford/es-module-shims/issues/228

**Comment by dhh** [comment]
Fixed via 0.9.3.
